pygsp/operators/reduction.py

```diff
@@ -12,17 +12,26 @@
 logger = build_logger(__name__)
 
 
-def graph_sparsify(M, epsilon, maxiter=10):
+def graph_sparsify(M, epsilon, maxiter=10, fast=True):
```

Comment:
I would set fast=False by default, so that the previous default behavior of this function remains unchanged. What do you think?